benchmarks/graphics_pipeline/main.cpp

```diff
@@ -189,6 +204,124 @@ void ProjApp::Setup()
         mCamera.SetPerspective(60.f, GetWindowAspect());
     }
 
+    // Uniform buffers
+    {
+        grfx::BufferCreateInfo createInfo        = {};
+        createInfo.size                          = 256;
```

Comment:
is this supposed to be 4x4x4=64? a bit error-prone if you hard code the value here in case someone changes the struct TransformData in the future. maybe better to use sizeof(TransformData) instead

Reply:
Done.

Reply:
i see, sure, but please add some comments here to explain why it is 256.

Reply:
I updated the value from 256 to PPX_MINIMUM_UNIFORM_BUFFER_SIZE ( = 256), I think with this change things are self explained.